### DISCUSSION TITLE
What IAM permissions are required for AWS?

### DISCUSSION THREAD

**Discussion by Harrolee** [skypilot-org/skypilot#2985] [Q&A] [answered]
Hey! 
Using SkyPilot would be easier if I knew upfront which policies to assign to the AWS IAM user that skypilot leverages. Can you tell me the permissions required? 
I could also figure out which permissions I need if I had a list of all of the actions that skypilot tries to execute.
Thanks!

**Comment by concretevitamin** [comment, marked as answer]
Hello @Harrolee - check out https://skypilot.readthedocs.io/en/latest/cloud-setup/cloud-permissions/aws.html

**Reply by Harrolee** [reply]
This is amazing! I'm not sure how I missed this. Thank you!!
